src/cargo/core/compiler/job_queue/mod.rs

```diff
@@ -806,9 +806,14 @@ impl<'cfg> DrainState<'cfg> {
             // `display_error` inside `handle_error`.
             Some(anyhow::Error::new(AlreadyPrintedError::new(error)))
         } else if self.queue.is_empty() && self.pending_queue.is_empty() {
+            let profile_link =
+                cx.bcx.config.shell().err_hyperlink(
+                    "https://doc.rust-lang.org/cargo/reference/profiles.html#profiles",
+                );
             let message = format!(
-                "{} [{}] target(s) in {}",
-                profile_name, opt_type, time_elapsed
+                "{}`{profile_name}` profile{} [{opt_type}] target(s) in {time_elapsed}",
```

Comment:
Not on topic, but we might way to make Hyperlink fancier like anstyle for open/close.

Reply:
I had considered that but didn't want to include it in this PR